lib/publiccloud/azure.pm

```diff
@@ -34,6 +35,10 @@ sub init {
     $self->provider_client->init();
 }
 
+sub az_arch {
```

Comment:
would be nice to add some description to this function

Reply:
done